.github/workflows/build.yml

```diff
@@ -112,11 +112,11 @@ jobs:
       - name: install tool chain
         if: steps.cache.outputs.cache-hit != 'true'
         run: |
-          sudo apt-get -y install libstdc++-10-dev clang-format-12 ccache lldb
+          sudo apt-get -y install libstdc++-10-dev clang-format-15 ccache lldb
```

Comment:
This won't work, `make format` will fail since clang-format-12 is not compatible with clang-format-15

Reply:
this is just an experiment, so make format doesn't matter here (I've disabled it and CI is already running)

Reply:
Ok Jammy runners are now available, so this should all be fixed